app/build.gradle

```diff
@@ -75,6 +75,12 @@ dependencies {
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$coroutineVersion"
 
     testImplementation 'junit:junit:4.13.2'
+    testImplementation "com.google.truth:truth:1.1.3"
```

Comment:
What do you need to use these dependencies ?

- com.google.truth:truth
- app.cash.turbine:turbine

Reply:
I used the Truth lib to make the assertThat in the tests. This lib turn the process to verify the asserts more easily.

The Turbine lib it's de only form that I found it to test Hot Flows and in this project I decide to use StateFlow instead LiveData.

Reply:
The `junit` not was sufficient ?

Nice decision about `Turbine` I had never seen before, I don't got were you using this.

Reply:
With junit I could use Assert.assertTrue method for example (Assert.assertTrue(getProductsResult.first() is ResultRemote.Success), but with truth when the test fails I can see this output on terminal that show me what is wrong.

expected instance of: com.example.mybeats.data.remote.responses.ResultRemote$Success
but was instance of : com.example.mybeats.data.remote.responses.ResultRemote$ErrorResponse$Unknown
with value          : Unknown(throwable=java.lang.Throwable)

With junit is it possible too?